### ISSUE TITLE
Support for Iridium Skyblock

### ISSUE THREAD

**Issue by srnyx** [endercrest/VoidSpawn#92]
**Describe the feature**
Add support for [Iridium Skyblock](https://www.spigotmc.org/resources/62480), a popular skyblock plugin

**Comment by QueenOfTheSea21** [comment]
Please do!!!! I currently use ASkyBlock for 1.12.2 but I'm updating the server (sometime) and will be using Iridium and would love to have this plugin work with it!

**Comment by endercrest** [comment]
There is a WIP version of this in: https://github.com/endercrest/VoidSpawn/tree/feature.iridium-skyblock. Looks like this plugin hasn't updated to 1.21.1 and so I will holding off on supporting this.

**Comment by srnyx** [comment]
> Looks like this plugin hasn't updated to 1.21.1 and so I will holding off on supporting this.

Yeah I no longer need support for it, not using either plugin anymore, thank you though
I'll keep the issue open in-case other people want it
